src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -130,7 +130,7 @@ def enum_default(resource_type, enum_name, enum_val_name):
         return None
 
 
-def three_state_flag(positive_label='true', negative_label='false'):
+def three_state_flag(positive_label='true', negative_label='false', invert=False):
```

Comment:
Please update the doc comment since it is no longer true that flag-like behavior always implies the affirmative. :)